src/script/descriptor.cpp

```diff
@@ -19,6 +19,97 @@
 
 namespace {
 
+////////////////////////////////////////////////////////////////////////////
+// Checksum                                                               //
+////////////////////////////////////////////////////////////////////////////
+
+// This section implements a checksum algorithm for descriptors with the following properties:
+// * Every 1 character substitition error counts as 1 or 2 symbol errors, but:
```

Comment:
Subsitute this substitition with substitution.

Reply:
Fixed.